tethys_portal/settings.py

```diff
@@ -37,8 +37,8 @@
 from tethys_portal.optional_dependencies import optional_import, has_module
 
 # optional imports
-bokeh_settings, bokehjsdir = optional_import(
-    ("settings", "bokehjsdir"), from_module="bokeh.settings"
+bokeh_settings, bokehjs_path = optional_import(
+    ("settings", "bokehjs_path"), from_module="bokeh.settings"
 )
```

Comment:
```suggestion
bokeh_settings = optional_import("settings", from_module="bokeh.settings")
```

Reply:
One of the reasons this is breaking now is because we were originally importing `bokehjsdir` incorrectly. It is imported into the `settings` module, but not indented to be imported from there. Rather, a method with the same name is defined on the `settings` object defined in the `settings` module. What we should have been doing is just importing ` bokeh_settings` and then calling `bokehjsdir` like this:

```python
bokeh_settings.bokehjsdir()
```
If we had done it that way it would still work and show the deprecation warning instead of raising an import error.

All of this applies to the new method `bokehjs_path`. It should be called on `bokeh_settings` and not imported directly.